src/pages/Signin/Signin.js

```diff
@@ -1,5 +1,72 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
 
 export default function Signin() {
-  return <div>Signin</div>;
+  return (
+    <div>
+      <Container>
+        <JoinWrap>
+          <JoinBox>
+            <Title>로그인</Title>
+            <KakaoBtnImg src="images/Signin/kakao_login_medium_wide.png"></KakaoBtnImg>
+            <IsSigninOrSignup>
+              아직 벅벅 계정이 없으신가요?
+              <Link to="/Signup">
+                <LoginSpan>회원가입</LoginSpan>
+              </Link>
```

Comment:
```suggestion
                            <SignupLink to="/Signup">회원가입</SignupLink>
```
불필요한 depth라고 보여지네요!
Link 컴포넌트로 사용해주시고, 스타일 컴포넌트에 적용하는 방법은 아래 스타일 부분을 참고해주세요!

그리고 조금 더 직관적인 컴포넌트 이름을 지어주세요!